src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands.py

```diff
@@ -719,3 +718,104 @@ def test_sb_migration(self, resource_group):
 
         # Delete Namespace - Premium
         self.cmd('servicebus namespace delete --resource-group {rg} --name {namespacenamepremium}')
+
+    @ResourceGroupPreparer(name_prefix='cli_test_sb_network')
+    def test_sb_network(self, resource_group):
+        self.kwargs.update({
+            'loc': 'westus',
+            'rg': resource_group,
+            'namespacename': self.create_random_name(prefix='servicebus-cli', length=20),
+            'namespacenamekafka': self.create_random_name(prefix='servicebus-cli1', length=20),
+            'tags': {'tag1=value1'},
+            'tags2': {'tag2=value2'},
+            'sku': 'Premium',
+            'tier': 'Premium',
+            'authoname': self.create_random_name(prefix='cliAutho', length=20),
+            'defaultauthorizationrule': 'RootManageSharedAccessKey',
+            'accessrights': 'Send',
+            'accessrights1': 'Listen',
+            'primary': 'PrimaryKey',
+            'secondary': 'SecondaryKey',
+            'isautoinflateenabled': 'True',
+            'maximumthroughputunits': 4,
+            'maximumthroughputunits_update': 5,
+            'varfalse': 'false',
+            'ipmask1': '1.1.1.1',
+            'ipmask2': '2.2.2.2',
+            'subnet1': '',
+            'subnet2': ''
+        })
+
+        from azure.cli.core.commands.client_factory import get_subscription_id
+        # update sub ids
+        self.kwargs.update({'subnet1': '/subscriptions/' + get_subscription_id(
+            self.cli_ctx) + '/resourcegroups/v-ajnavtest/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/default'})
+        self.kwargs.update({'subnet2': '/subscriptions/' + get_subscription_id(
+            self.cli_ctx) + '/resourcegroups/v-ajnavtest/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/sbdefault'})
```

Comment:
These hard-coded resource groups (v-ajnavtest) will not re-record generally and seems to depend on pre-existing resources. You should (ideally) use the resource group the test generates.

Reply:
updated code to create VNet and Subnet